doc/tutorials/acrn-sanity-test.rst

```diff
@@ -0,0 +1,347 @@
+
+ST_HV_Memory_LaaG_Default_Partition_Check
+*****************************************
+
+* Description
+
+This test case focus on checking if the pass through memory to Guest OS and 
+check if the memory size is in a reasonable scope as launch_uos.sh defined. 
+Since the Guest OS will take a part of memory, the available memory size will 
+not be exact as what pass-through.
+
+* Prerequisites
+
+Service OS and Guest OS boot up normally
+
+* Test Steps
+
+1. Boot to SOS
+#. Boot LaaG UOS with default memory setting in launch_uos.sh
+#. Check ``cat /proc/meminfo`` and see if the available memory is close the size passed through to UOS
+
+Expected Result
+
+1. Boot to UOS successfully
+#. Step 3: UOS - Depends on parameter passed to acrn-dm in launch_uos.sh.
+
+ST_HV_DM_LaaG_DefaultCPU_2GMemory_Conf
+**************************************
+
+* Description
+
+This test case focus on check if the pass through memory and CPU with certain 
+value in launch_uos.sh are correct.
+
+* Prerequisites
+
+Service OS and Guest OS boot up normally
+
+* Test Steps
+
+1. Boot to SOS
+#. Create UOS with 1 core and memory=2048MB
+#. Check ``cat /proc/cpuinfo`` and ``cat /proc/meminfo``
+
+* Expected Result
+
+1. Boot to UOS successfully
+#. Step 3: UOS memory size and CPU cores depends on parameter passed to acrn-dm in launch_uos.sh.
+
+IO
+**
+
+ST_ACRNIO_LaaG_Virtio_net
+*************************
+
+* Description
+
+This test case focus on checking if Service OS and LaaG can be allocated different IP, 
+and ping is ok by sharing same physical network card.
+
+* Prerequisites
+
+Need support Virtio, and bridge/tap is created normally in Service OS
+
+* Test Steps
+
+1. Boot the Device Under Test (DUT), and enter SOS from ACRN hypervisor
+
+   .. code-block:: none
+
+      $ vm_console
+
+#. Check IP and bridge/tap after log in sos
+
+   .. code-block:: none
+     
+      $ ip a
+      $ brctl show
+      bridge name     bridge id               STP enabled     interfaces
+      acrn-br0         8000.1ad4ac453b20       no              acrn_tap0
+                                                               ens4
+
+#. Boot UOS using the launch_uos.sh 
+
+   .. code-block:: none
+   
+      $ bash /usr/share/acrn/samples/nuc/launch_uos.sh
+
+#. Check UOS IP address
+
+* Expected Result
+
+IP address should be allocated for both SOS and UOS, and ping is ok
+
+ST_ACRNIO_LaaG_Virtio_storage
+*****************************
+
+* Description
+
+This test case focus on checking if LaaG using virtio storage can boot up successfully.
+
+* Prerequisites
+
+Need support Virtio
+
+* Test Steps
+
+1. In SOS: copy or create one img file as rootfs.
+#. Boot UOS using this img file in cmd line in launch_uos.sh.
+
+   .. code-block:: none
+
+   $ mkdir -p /home/clear/uos/
+
+   $ cd /home/clear/uos/
+
+   $ curl https://download.clearlinux.org/releases/xxxxx/clear/clear-xxxxx-kvm.img.xz -o uos.img.xz
+
+   $ unxz uos.img.xz
+
+* Expected Result
+
+UOS can boot up successfully
```

Comment:
What about the kernel modules, are these not needed?

Reply:
This test can be performed even if it is not replaced